index.js

```diff
@@ -1,39 +1,267 @@
+        stateThree();
+        fifthLoop = false;
+      } else if (answer5.includes("inv") || answer5 === "i") {
+        showInv();
+        answer5 = await ask("\n");
+      } else if (answer5.includes("take") && answer5.includes("hat")) {
+        inv.hat = "Bob's Elegant Hat";
+        answer5 = await ask(
+          "You have taken Bobs Elegant Coding SuperPowers Hat \n"
+        );
+      } else if (answer5.includes("take") && answer5.includes("cheat sheet")) {
+        inv.vtcs = "Vermont Tech Cheat Sheet";
+        answer5 = await ask("You have taken Sams Vermont Tech Cheat Sheet\n");
+      } else if (
+        answer5.includes("fix bug") &&
+        inv.vtcs == "Vermont Tech Cheat Sheet" &&
+        inv.hat == "Bob's Elegant Hat"
+      ) {
+        console.log(
+          "Using the combined superpowers of Bobs Elegant Hat and Sams Vermont Tech Cheat Sheet you have fixed the bug and won the game!\n"
+        );
+        fifthLoop = false;
+        break;
+        process.exit;
+      } else {
+        answer5 = await ask("Im sorry, I cannot " + answer5 + "\n");
+      }
+    }
+  }
+  /*
+FIFTH ROOM ENDS/SIXTH ROOM BEGINS ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  async function stateSix() {
+    //THIS WILL BE ETERNITY WEB
+    let sixthLoop = true;
+    let answer6 = await ask(
+      "Welcome to Eternity web. Maybe I should apply to work here? On the main desk you see a pile of change. \n"
+    );
+    while (sixthLoop === true) {
+      if (answer6.includes("hall")) {
+        stateThree();
+        sixthLoop = false;
+      } else if (answer6.includes("inv") || answer6 == "i") {
+        showInv();
+        answer6 = await ask("\n");
+      } else if (answer6.includes("take") || answer6.includes("change")) {
+        inv.change = "Change";
+        answer6 = await ask("You have taken the pile of change. \n");
+      } else if (answer6.includes("apply") || answer6.includes("job")) {
+        answer6 = await ask(
+          "They don't want to hire you. Perhaps you should go back to class and fix the bug!"
+        );
+      } else {
+        answer6 = await ask("Im sorry, I cannot " + answer6 + "\n");
+      }
+    }
+  }
+  //Sparechange Lady
+  async function stateSeven() {
+    let seventhLoop = true;
+    let answer7 = await ask(
+      "Sparechange lady looks mighty hungry. Do you have any change for her? \n"
+    );
+    while (seventhLoop === true) {
+      if (answer7 == "right") {
+        begin();
+        seventhLoop = false;
+      } else if (answer7 == "left") {
+        stateEight();
+        seventhLoop = false;
+      } else if (answer7.includes("inv") || answer7 === "i") {
+        showInv();
+        answer7 = await ask("\n");
+      } else {
+        answer7 = await ask("Im sorry, I cannot " + answer7 + "\n");
+      }
+    }
+  }
+  //Mirabelles
+  async function stateEight() {
```

Comment:
If you made your rooms objects instead of loops, and you made a class with a constructor for them, you could probably shorten this file by about 100 lines if you just call methods on the current room. You would also then need some way of tracking which room you're in, but it will be much more scale-able, and flexible than all these loops.